packages/Parallel/Command/WorkerCommandLineFactory.php

```diff
@@ -66,10 +65,6 @@ public function create(
         }
 
         $workerCommandArray[] = $workerCommandName;
-        if ($projectConfigFile !== null) {
```

Comment:
This value is always `null`. It's better resolved from `mirrorCommandOptions()` method. That duplicates all the options/arguments in main command.
`